test/e2e/kubectl.go

```diff
@@ -125,7 +125,7 @@ var _ = framework.KubeDescribe("Kubectl client", func() {
 	framework.KubeDescribe("Update Demo", func() {
 		var updateDemoRoot, nautilusPath, kittenPath string
 		BeforeEach(func() {
-			updateDemoRoot = filepath.Join(framework.TestContext.RepoRoot, "docs/user-guide/update-demo")
+			updateDemoRoot = filepath.Join(framework.TestContext.GeneratedAssetsDir, framework.GENERATED_ASSETS, "docs/user-guide/update-demo")
```

Comment:
Ohh I really like the Asset mechanics.. That's pretty much exactly what I think we want.